MOxUnit/assertEqual.m

```diff
@@ -26,14 +26,14 @@ function assertEqual(a, b, message)
 % NNO Jan 2014
 
 
-    if ~isequal(size(a), size(b))
-        whatswrong='inputs are not of the same size';
-        error_id='assertEqual:nonEqual';
-
-    elseif ~isequal(class(a), class(b))
+    if ~isequal(class(a), class(b))
         whatswrong='inputs are not of the same class';
         error_id='assertEqual:classNotEqual';
 
+    elseif ~isequal(size(a), size(b))
```

Comment:
I see your point, but the original Matlab xunit framework checks for equal class first. I prefer to keep it compatible with that when possible. A comment would be useful though to explain the order.